### PR TITLE
more tests for multivalue binary operations

### PR DESCRIPTION
**Pull request by rederrik**
Added a test for multivalue binary copy.
Added test to check unusual binary properties behavior: one for multivalue property with no values in it and another for multivalue property with a single value in it - it fails with error.

### CODE REVIEW

**Comment by dbu** [comment]
sorry, for being unresponsive. i am still travelling. i will have a look at this next week unless @lsmith77 beats me to it.

**Comment by lsmith77** [comment]
these tests seem to break with Jackrabbit https://github.com/jackalope/jackalope-jackrabbit/pull/122

File: fixtures/10_Writing/copy.xml [hunk 1]
```diff
@@ -502,4 +502,22 @@
...
+                <sv:value>nt:unstructured</sv:value>
+            </sv:property>
+            <sv:node sv:name="multiple">
+                <sv:property sv:name="jcr:primaryType" sv:type="Name">
+                    <sv:value>nt:unstructured</sv:value>
+                </sv:property>
+                <sv:property sv:name="jcr:data" sv:type="Binary" sv:multiple="true"><sv:value>RnVzY2UgZmVybWVudHVtLiBBbGlxdWFtIGxvYm9ydGlzLiBNYXVyaXMgdHVycGlzIG51bmMsIGJsYW5kaXQgZXQsIHZvbHV0cGF0IG1vbGVzdGllLCBwb3J0YSB1dCwgbGlndWxhLiBQZWxsZW50ZXNxdWUgYXVjdG9yIG5lcXVlIG5lYyB1cm5hLiBQcm9pbiBtYWduYS4gQ3VyYWJpdHVyIHVsbGFtY29ycGVyIHVsdHJpY2llcyBuaXNpLiBQcmFlc2VudCBjb25ndWUgZXJhdCBhdCBtYXNzYS4gTmFtIGFkaXBpc2NpbmcuIE51bGxhIHBvcnRhIGRvbG9yLiBGdXNjZSBjb21tb2RvIGFsaXF1YW0gYXJjdS4=</sv:value><sv:value>RnVzY2UgZmVybWVudHVtLiBBbGlxdWFtIGxvYm9ydGlzLiBNYXVyaXMgdHVycGlzIG51bmMsIGJsYW5kaXQgZXQsIHZvbHV0cGF0IG1vbGVzdGllLCBwb3J0YSB1dCwgbGlndWxhLiBQZWxsZW50ZXNxdWUgYXVjdG9yIG5lcXVlIG5lYyB1cm5hLiBQcm9pbiBtYWduYS4gQ3VyYWJpdHVyIHVsbGFtY29ycGVyIHVsdHJpY2llcyBuaXNpLiBQcmFlc2VudCBjb25ndWUgZXJhdCBhdCBtYXNzYS4gTmFtIGFkaXBpc2NpbmcuIE51bGxhIHBvcnRhIGRvbG9yLiBGdXNjZSBjb21tb2RvIGFsaXF1YW0gYXJjdS4=</sv:value>
```
Comment (dbu) [inline fixtures/10_Writing/copy.xml]: can you just add two values here to make sure jackrabbit gets it correctly? lets put one `<sv:value>` per line.

Comment (dbu) [inline fixtures/10_Writing/copy.xml]: a sorry, its 2 already. can you just reformat to make it readable please?

**Comment by dbu** [comment]
about to wrap this up. see #170

**Pull request closed without merging**